modules/eventing/pages/eventing-advanced-keyspace-accessors.adoc

```diff
@@ -516,6 +537,10 @@ The positional parameter (denoted by _meta_ in the prototype above) represents t
 +
 The key of the document to be used for the operation. This is a mandatory parameter and must be of JavaScript _string_ type. If the specified key is not present in the bucket, the operation fails and _key_not_found_ will be set in the error object.
 
+** _meta.keyspace (type: Object)_
++
+The keyspace of the document to be used for the operation with the JavaScript format "keyspace":{"bucket_name":_string_,"scope_name":_string_,"collection_name":_string_}. This parameter is only required if you are using a a _binding_ with a wildcard for the scope and/or the collection. [.status]#Couchbase Server 7.1.1#
```

Comment:
```suggestion
The keyspace of the document to be used for the operation with the JavaScript format "keyspace":{"bucket_name":_string_,"scope_name":_string_,"collection_name":_string_}. 
This parameter is only required if you are using  a _binding_ with a wildcard for the scope and/or the collection. [.status]#Couchbase Server 7.1.1#
```

As a general comment, it's usually a good idea to have repeated text set up as an Asciidoc attribute, instead of copying and pasting.